app/views/pages/rules_of_origin_duty_drawback.html.erb

```diff
@@ -0,0 +1,93 @@
+<% content_for :top_breadcrumbs do %>
```

Comment:
Can you add the sidebar link in the glossary page to this page

https://github.com/trade-tariff/trade-tariff-frontend/blob/main/app/views/pages/glossary/index.html.erb#L48